Configuration menu

agriffard opened the pull request:
Sort most menu items under Configuration by alphabetic order and Fix Google icon.

Before:
![image](https://user-images.githubusercontent.com/703248/68528360-eb85a680-02f1-11ea-83d0-0118a94cbe5a.png)


After:
![image](https://user-images.githubusercontent.com/703248/68529363-21308c80-02fe-11ea-8545-7c88b8eaaf24.png)

agriffard commented:
We could also choose to put the Configuration menu at the bottom of the admin menu.

![image](https://user-images.githubusercontent.com/703248/68528400-8ed6bb80-02f2-11ea-97bd-eedaa53cd4ee.png)

Does anyone think, like me, that it would be a good idea?

deanmarcussen commented:
> Does anyone think, like me, that it would be a good idea?

Yes, I agree, configuration is better at the bottom.

And also like the sorting by alpha, makes it much easier to create new items. and just makes sense.

But
- maybe can we keep it to Google Analytics - makes it clearer what it is.
- if you used `T["Tenants"]` for the sort, then it would sort by translation

agriffard commented:
Good suggestions @deanmarcussen :
Moved Configuration at bottom of the menu.
Moved `Facebook App` and `Google Analytics` to Configuration > Settings menu.
Wrapped titles with T[""].

See the new After screenshot.

deanmarcussen commented on src/OrchardCore.Modules/OrchardCore.Features/AdminMenu.cs:
```diff
@@ -22,7 +22,7 @@ public Task BuildNavigationAsync(string name, NavigationBuilder builder)
             builder
-                .Add(T["Configuration"], "4", configuration => configuration
+                .Add(T["Configuration"], "9", configuration => configuration
                     .AddClass("menu-configuration").Id("configuration")
```
Nice one @agriffard 

One last thing I notice, this could be 100 everywhere, so that it always stays at the bottom. 
Even when we add new menu items.
Other places it is 10, here it is 9. Or maybe it just needs to be 100 once, somewhere on a feature that is always on. and never specified again.

Or maybe it should be set on `Settings` the once. That would always be on? I think

agriffard replied:
ok @deanmarcussen Configuration = "100"

Sorry, I didn't remember I switched back my branch to dev so I already pushed it to dev but I guess it doesn't harm.

agriffard commented:
@deanmarcussen 
The difficulty for some menus like Configuration and Search is indeed to find a "main" feature on which all other sub features depend on.

hishamco commented:
> Yes, I agree, configuration is better at the bottom.

+1

deanmarcussen commented on src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/AdminMenu.cs:
```diff
@@ -24,7 +24,7 @@ public Task BuildNavigationAsync(string name, NavigationBuilder builder)
             builder
                 .Add(S["Configuration"], "10", design => design
                     .AddClass("menu-configuration").Id("configuration")
```
> The difficulty for some menus like Configuration and Search is indeed to find a "main" feature on which all other sub features depend on.

@agriffard agreed, this is the dificulty! so maybe the answers is to do it everywhere? because mostly these are going to be cut and pasted into new menus, so good to get the cut and paste code right.

Also noting that it must be last module registration wins, because here it is `10` and yet `100` is winning
